src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -4,7 +4,6 @@
     (function (options) {
         // Pre check
         var statusElement = document.querySelector('#bankIdLoginStatus');
-        var statusWrapperElement = statusElement.querySelector('.status-wrapper');
```

Comment:
Just realized we need to add back the `.status-wrapper` class at the `.card` element, and modify the JS back to apply styling etc on that element.